take2.js

```diff
@@ -0,0 +1,47 @@
+//THIS IS THE FILE TO BE GRADED.  
+/*I am so overwhelmed.  I thought I knew what I was doing, but obviously not. Tried lots of different things.  This is as far as I could get. */
+
+//ADD a var XX = board.map....to make a new array
+
+
+var Board = function (fromPeg, toPeg) {
```

Comment:
I don't think you need to define these parameters (`fromPeg and `toPeg`) here. You would use the `Board` function just to construct your game in the beginning. In fact, since your actual board is defined on line 8, this function could probably be called something like `Game` or `TowerGame`. You would use it like this:

```js
var myGame = TowerGame(); // myGame would now be an object as returned from invoking `TowerGame`

myGame.moveDisk(0, 2); // perhaps move is a function that actually attempts to move a disc from the first peg to the last one
```